File: app/src/main/java/co/tpcreative/supersafe/model/Cover.java
package co.tpcreative.supersafe.model;

import co.tpcreative.supersafe.common.entities.ItemEntity;
import co.tpcreative.supersafe.common.entities.MainCategoryEntity;

public class Cover {

    public Categories categories;
    public MainCategoryEntity mainCategories;
    public ItemEntity items;
    public boolean isSelected;

    public int getCategoryId() {
        return categories.id;
    }

    public String getCategoryName() {
        return categories.name;
    }


}

File: app/src/main/java/co/tpcreative/supersafe/model/Cover.kt
package co.tpcreative.supersafe.model
import co.tpcreative.supersafe.common.entities.ItemEntity
import co.tpcreative.supersafe.common.entities.MainCategoryEntity

class Cover {
    var categories: Categories? = null
    var mainCategories: MainCategoryEntity? = null
    var items: ItemEntity? = null
    var isSelected = false
    fun getCategoryId(): Int? {
        return categories?.id
    }

    fun getCategoryName(): String? {
        return categories?.name
    }
}